libavpipe/src/avpipe_xc.c

```diff
@@ -2391,8 +2320,9 @@ do_bypass(
             packet->pos, packet->size, packet->stream_index,
             packet->flags, packet->data);
     } else {
-        if (av_interleaved_write_frame(format_context, packet) < 0) {
-            elv_err("Failure in copying bypass packet xc_type=%d, url=%s", p->xc_type, p->url);
+        int rc = av_interleaved_write_frame(format_context, packet);
+        if (rc < 0) {
+            elv_err("Failure in copying bypass packet xc_type=%d rc=%d url=%s", p->xc_type, rc, p->url);
```

Comment:
Could also use `av_err2str` here to skip interpreting the return code (macro from ffmpeg error.h)